scripts/apps/search/components/SmoothLoader.tsx

```diff
@@ -55,7 +55,9 @@ export class SmoothLoader extends React.Component<IProps> {
                 {
                     loading && (
                         <div style={{height: '100%', position: 'relative'}}>
-                            <ListItemLoader />
+                            <div data-test-id="loading-placeholder">
+                                <ListItemLoader />
+                            </div>
```

Comment:
Since you wrap this component in a div with the same data-test-id wouldn't it be better if we put the wrapper inside the component itself? Note: the component is in superdesk-ui-framework.